### ISSUE TITLE
The calendar is displaying at reduced width and height to the full window

### ISSUE THREAD

**Issue by SiXiTH-git** [702573N/Obsidian-Tasks-Calendar#61]
I don't know a lot about CSS so I don't really know how to fix this on my own, or how to even figure out why its displaying like this, but its like the padding on the left and right are set really high, or maybe its stuck displaying in a mobile phone format? I am honestly not sure either way, but will leave a screen shot for visual representation.
![Screenshot 2023-02-16 173606](https://user-images.githubusercontent.com/18738425/230816813-179b9369-e898-4330-87f8-b7d6237c2734.png)

**Comment by OGKevin** [comment]
![Screenshot 2023-04-14 at 15 54 37](https://user-images.githubusercontent.com/17928966/232063847-537f44e8-1d39-4355-874d-d6e55786b01a.png)
Turn off `Readible line lenght`

**Comment by bhoffmann93** [comment]
Does anyone know of a way to display only the full width calendar and not each note?

**Comment by giulioungaretti** [comment]
yes @bhoffmann93 !  Try following here: https://gist.github.com/vii33/f2c3a85b64023cefa9df6420730c7531 !

What worked for me was the following, but only with the default theme! Some other tweaks are needed for minimal theme but I gave up instead!


custom css snippet:
```css
.wide {
	--file-line-width: 99%;
	padding: 10;
}
```

and then a yaml frontmatter

``` yaml
---
cssclass: wide
---
```
